pressbooks/pressbooks-aldine
# Add classes for nav primary menu items and footer items

Issue: https://github.com/pressbooks/pressbooks-book/issues/781

Related PR: https://github.com/pressbooks/pressbooks-book/pull/821

This change adds 2 new classes for each top menu items:
![Screenshot from 2021-07-27 10-46-03](https://user-images.githubusercontent.com/13248424/127175166-76ea681a-b45b-4203-a412-d8141a8990d4.png)
and for footer menu items:
![Screenshot from 2021-07-27 10-46-30](https://user-images.githubusercontent.com/13248424/127175255-bc7c429c-b899-448d-bfb5-90e812d18d98.png)
with the objective to allow users to customize the style.
Classes added:
For top menu:
```
nav--primary-item (for top menu items customization)
nav--primary-item-home (home item)
nav--primary-item-contact (contact item)
nav--primary-item-my-books (My Books item)
nav--primary-item-sign-out (Sign Out item)
nav--primary-item-sign-in (Sign In Item)
```

For footer menu:
```
footer__pressbooks__links__list-item (for footer menu items)
footer__pressbooks__links__list-item-guide-tutorials (Guide and Tutorials item)
footer__pressbooks__links__list-item-contact (Contact item)
```

### Test notes
- Make sure you have this changes + changes in https://github.com/pressbooks/pressbooks-book/pull/821
- Log in as Admin and go to Appearance > Customize > Additional CSS
- Add different styles for the classes mentioned above and confirm users can add personalized styles to items.
